build.properties

```diff
@@ -18,7 +18,7 @@ phpcs.prepush.enable = 1
 phpcs.files = ${website.modules.dir};${website.profiles.dir};${website.themes.dir};${behat.dir}/src
 
 # Paths to ignore, delimited by semicolons.
-phpcs.ignore = ${website.modules.dir}/contrib;${website.themes.dir}/joinup/prototype
+phpcs.ignore = ${website.modules.dir}/contrib;${website.themes.dir}/joinup/prototype;${website.settings.local.php.example};${website.settings.dir}/default.settings.php;${website.drupal.dir}/autoload.php
```

Comment:
This was fixed in the Git pre-push hook https://gist.github.com/claudiu-cristea/1ec87c25930861654cc94251da69f14b/revisions